extension/manifest.json

```diff
@@ -15,6 +15,7 @@
     "experienceSamplingPrivate",
     "notifications",
     "storage",
+    "tabs",
```

Comment:
That's OK, because I actually need to forcibly uninstall the extension from all the current dogfooders before we start running the real trial.